+ date
+ echo MONITOR: START at Fri Aug 26 23:55:42 UTC 2016
MONITOR: START at Fri Aug 26 23:55:42 UTC 2016
+ python=python
+ input_file=./static/upload/honghuangzhili.jpg
+ style_file=./static/upload/picasso.jpg
+ output_name=honghuangzhili_picasso
+ uniq_id=20160827_07:55:43_honghuangzhili_picasso_20
+ max_num_epochs=20
+ mkdir -p output/honghuangzhili_picasso/
+ /bin/rm -rf output/honghuangzhili_picasso/honghuangzhili_picasso_200.jpg
+ output_file=output/honghuangzhili_picasso/honghuangzhili_picasso_20.jpg
+ python run.py --content-image ./static/upload/honghuangzhili.jpg --style-image ./static/upload/picasso.jpg --gpu -1 --save-epochs 5 --max-num-epochs 20 --output output/honghuangzhili_picasso/honghuangzhili_picasso_20.jpg
libdc1394 error: Failed to initialize libdc1394
Traceback (most recent call last):
  File "run.py", line 96, in <module>
    content_np = PreprocessContentImage(args.content_image, args.max_long_edge)
  File "run.py", line 49, in PreprocessContentImage
    img = io.imread(path)
  File "/usr/local/lib/python2.7/dist-packages/skimage/io/_io.py", line 61, in imread
    img = call_plugin('imread', fname, plugin=plugin, **plugin_args)
  File "/usr/local/lib/python2.7/dist-packages/skimage/io/manage_plugins.py", line 211, in call_plugin
    return func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/skimage/io/_plugins/pil_plugin.py", line 35, in imread
    with open(fname, 'rb') as f:
IOError: [Errno 2] No such file or directory: './static/upload/honghuangzhili.jpg'
+ date
+ echo MONITOR: END at Fri Aug 26 23:55:43 UTC 2016
MONITOR: END at Fri Aug 26 23:55:43 UTC 2016
